uberfire-widgets/uberfire-widgets-service/uberfire-widgets-service-api/src/main/java/org/uberfire/ext/services/shared/preferences/UserPreference.java

```diff
@@ -0,0 +1,26 @@
+package org.uberfire.ext.services.shared.preferences;
+
+import org.jboss.errai.common.client.api.annotations.Portable;
+
+@Portable
+public class UserPreference {
+    UserPreferencesType type;
```

Comment:
Should this be "package" access level; private or protected may be more suitable.